initrd/bin/cbfs-init

```diff
@@ -9,12 +9,20 @@ if [ -z "$CONFIG_PCR" ]; then
 	CONFIG_PCR=7
 fi
 
+if [ "$CONFIG_CBFS_VIA_FLASHROM" = "y" ]; then
+	# Use flashrom directly, because we don't have /tmp/config with params for flash.sh yet
+	/bin/flashrom -p internal --fmap -i COREBOOT -i FMAP -r /tmp/cbfs-init.rom > /dev/null 2>&1 \
```

Comment:
Ideally we implement something that can reuse FLASH_OPTIONS fromm other boards configs and uses flashprog as of master now. What is this 16MB limitation from CBFS?